src/components/shared/NavBar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="navbar">
+    <nav>
+      <ul>

```

Comment:
This is more complicated than it needs to be. You can get rid of the <ul> and <li> elements. You also don't need to add the "sep" lines manually. Look at what was removed from the home page for how to do this in a simpler way.